tests/d3d12.c

```diff
@@ -44639,6 +44639,158 @@ static void test_texture_feedback_instructions_dxil(void)
     test_texture_feedback_instructions(true);
 }
 
+static void test_aliasing_barrier(void)
```

Comment:
What exactly do we count as "success" here? Not crashing / no validation errors?

Reply:
Yes, there isn't much meaningful we can test for here otherwise.

Reply:
Added a comment to clarify intent of the test.